srml/contract/src/lib.rs

```diff
@@ -280,15 +317,16 @@ decl_module! {
 			result.map(|_| ())
 		}
 
-		/// Create a new contract, optionally transferring some balance to the created account.
+		/// Creates a new contract from the `codehash` generated by PUT_CODE, optionally transferring some balance.
 		///
 		/// Creation is executed as follows:
 		///
 		/// - the destination address is computed based on the sender and hash of the code.
-		/// - account is created at the computed address.
+		/// - the smart-contract account is created at the computed address.
 		/// - the `ctor_code` is executed in the context of the newly created account. Buffer returned
 		///   after the execution is saved as the `code` of the account. That code will be invoked
-		///   upon any message received by this account.
+		///   upon any call received by this account.
+		/// - the contract is initialized
```

Comment:
Same for all points here.
```suggestion
		/// - The contract is initialized.
```